BraintreeCore/src/main/java/com/braintreepayments/api/core/AnalyticsClient.kt

```diff
@@ -27,10 +29,25 @@ internal class AnalyticsClient(
     private val applicationContext: Context
         get() = merchantRepository.applicationContext
 
-    fun sendEvent(event: AnalyticsEvent) {
+    fun sendEvent(
+        eventName: String,
+        analyticsEventParams: AnalyticsEventParams = AnalyticsEventParams()
```

Comment:
Takes in `AnalyticsEventParams` instead of `AnalyticsEvent` which is internal to the core module.